apps/next/package.json

```diff
@@ -6,19 +6,20 @@
   },
   "private": true,
   "scripts": {
-    "build:next": "next build",
-    "build:oci": "nixpacks build ./ --name metal-next --start-cmd 'bun run start:next'",
+    "build:next": "NEXT_TELEMETRY_DISABLED=1 next build",
+    "build:oci": "NEXT_TELEMETRY_DISABLED=1 nixpacks build ./ --name metal-next --start-cmd 'bun run start:next'",
```

Comment:
couldn't these be set these in the GHA env: section? It's confusing to see these here and also have two other places we put env:
- the .env file 
- the env: section of a GHA

Reply:
Yes I will set it in an env var and expose that value to all GHAs.

Reply:
Actually, I don't think it needs to be a var since its value won't change- unless you have some scenarios in mind in which you'll like to enable this.

Reply:
Already explained on Discord but putting this here for the future:

> Ah sorry I didn't explain myself very well. I wasn't advocating to keep NEXT_TELEMETRY_DISABLED in the scripts section. No, no. I meant to say should I write it like this in the GHA:
> ```
>     env:
>       NEXT_TELEMETRY_DISABLED: 1
> ```
> 
> or
> ```
>     env:
>       NEXT_TELEMETRY_DISABLED: ${{ env.NEXT_TELEMETRY_DISABLED }}
> ```